jberet-se/src/main/java/org/jberet/se/BatchSEEnvironment.java

```diff
@@ -103,6 +111,33 @@ protected int getMaximumPoolSize() {
         this.jobXmlResolver = new ChainedJobXmlResolver(userJobXmlResolvers, DEFAULT_JOB_XML_RESOLVERS);
     }
 
+
+    /**
+     * @param propVal The propVal to process. For example: `${DB_PASS:xyz}`, this method will try to
+     *                fetch the environment variable `DB_PASS` firstly, and if it does not exist, the method will
+     *                return the default value `xyz`.
+     * @return Get the value of the `propVal`.
```

Comment:
avoid using `xxx` in javadoc; use {@code xxx} format instead.

Move the description to the top of the method javadoc, and keep for the param description: @param propVal the propVal to process.

I see this method is only used by 2 test methods, not in production code. Can you double check?

Reply:
Also can you check if this method can be made package private, instead of public?